template.rb

```diff
@@ -167,6 +168,24 @@ def setup_timezone
   gsub_file("config/application.rb", "Central Time (US & Canada)", "Adelaide")
 end
 
+# Use rails_semantic_logger to log to stdout in JSON format
+def setup_logger
+  gem("rails_semantic_logger")
+
+  gsub_file("config/environments/production.rb", /# Log to STDOUT by default.*\n/,
+            ["# Configure logging as JSON to stdout",
+             "STDOUT.sync = true",
+             "config.rails_semantic_logger.add_file_appender = false",
+             "config.semantic_logger.add_appender(io: STDOUT, formatter: :json)"]
+              .join("\n  "))
+
+  gsub_file("config/environments/production.rb", /# Prepend all log lines.*\n/, "")
+  gsub_file("config/environments/production.rb", /config.log_tags.*\n/, "")
+  gsub_file("config/environments/production.rb", /config.logger.*\n/, "")
+  gsub_file("config/environments/production.rb", /\.tap\s* \{ \|logger\|.*\n/, "")
+  gsub_file("config/environments/production.rb", /\.then \{ \|logger\|.*\n/, "")
```

Comment:
I'd prefer to see this written as a single gsub, both from a safety point of view (don't accidentally remove something else) and from a developer-friendliness point of view. If that's not possible then this is ok, but please add a comment explaining what the source pattern looks like.